app/assets/stylesheets/base/_mixins.scss

```diff
@@ -21,6 +21,23 @@
 	}
 }
 
+@mixin themed($prop: background-color, $map-key: "background") {
+	#{$prop}: theme-light($map-key);
+
+	body.dark & {
+		#{$prop}: theme-dark($map-key);
+	}
+}
+
+@mixin themed-with-cards($prop: background-color, $map-key: "background") {
+	body.with-cards & {
+		#{$prop}: theme-light($map-key);
+	}
+
+	body.dark.with-cards & {
+		#{$prop}: theme-dark($map-key);
+	}
+}
```

Comment:
this is just a general comment about modifiers, largely based on:
- [bulma's modifiers](https://bulma.io/documentation/modifiers/syntax/)
- but also just my exp in doing them in a BEM way

using `is-` and `has-` just makes things a little easier to read.
so i'd do `.is-dark` and `.has-cards` instead of `.dark` and `.with-cards`

it's "just another one of those opinionated things" that makes naming things a little more consistent. 

not suggesting you redo it (specially if a simple replace-all won't do the trick), but still worth noting here.

Reply:
noted. I'll see about replacing after merge

Reply:
noted. I'll do a find and replace, shouldn't be that bad